ignite/metrics/ssim.py

```diff
@@ -85,11 +85,13 @@ def __init__(
 
     @reinit__is_reduced
     def reset(self) -> None:
-        self._sum_of_batchwise_ssim = 0.0  # Not a tensor because batch size is not known in advance.
+        self._sum_of_batchwise_ssim = (
```

Comment:
Now, `self._sum_of_batchwise_ssim` is defined as a tuple, right ? Can't we keep it as float ?

Reply:
yeah, it is still float, but `black` just formatted it. To be a single value tuple it needs to be written with a comma
```python
self._sum_of_batchwise_ssim = (0.0, )
```
I will move the comment just above the attribute assignment

Reply:
I mean, previously it was a plain float, not tuple and now it becomes a tuple with a single float:
```diff
- self._sum_of_batchwise_ssim = 0.0
+ self._sum_of_batchwise_ssim = (0.0, )
```
It is not intended, right ?

Reply:
sorry, what I meant
`0.0 = (0.0) != (0.0, )`
So it is still a float and not a tuple 😄

Reply:
Thanks ! Never got that `0.0 = (0.0)` :)

Reply:
I have seen it many times, when I started to use `black`. I usually used that syntax for multiline strings.